skull-king-app/src/components/PlayerStatusCard/PlayerStatusCard.tsx

```diff
@@ -80,36 +115,102 @@ export const PlayerStatusCard = (props: PlayerStatusCardProps) => {
+          show={dropdownOpen && allowBonus}
+          onToggle={(isOpen) => setDropdownOpen(isOpen)}
         >
-          <div
-            style={{ "--width": "75px" } as React.CSSProperties}
-            className="numberDisplayContainer"
+          <Dropdown.Toggle 
+            variant="outline-primary" 
+            className="bonusDropdownToggle"
+            disabled={!allowBonus}
           >
-            {currentBonus}
-          </div>
+            Select bonus…
+          </Dropdown.Toggle>
+          
+          <Dropdown.Menu className="bonusDropdownMenu">
+            {availableOptions.length > 0 ? (
+              availableOptions.map((option) => (
+                <Dropdown.Item
+                  key={option.id}
+                  onClick={(e) => {
+                    e.preventDefault();
+                    e.stopPropagation();
+                    handleBonusSelect(option.id);
+                  }}
+                  className="bonusDropdownItem"
+                >
+                  {option.label} (+{option.points})
+                </Dropdown.Item>
+              ))
+            ) : (
+              <Dropdown.Item disabled className="bonusDropdownItem">
+                All bonuses selected
+              </Dropdown.Item>
+            )}
+            <Dropdown.Divider />
+            <Dropdown.Item
+              onClick={(e) => {
+                e.preventDefault();
+                e.stopPropagation();
+                setDropdownOpen(false);
+              }}
+              className="bonusDropdownCloseItem"
+            >
+              Close
+            </Dropdown.Item>
+          </Dropdown.Menu>
+        </Dropdown>
+        
+        <div className="selectedBonusesDisplay">
+          {selectedBonuses.length > 0 ? (
+            <div className="bonusListContainer">
+              <div className="bonusHeader">Selected Bonuses:</div>
+              {selectedBonuses.map((bonusId, index) => {
+                const option = BONUS_OPTIONS.find(opt => opt.id === bonusId);
+                return (
+                  <div key={`${bonusId}-${index}`} className="selectedBonusItem">
+                    <span>{option?.label} (+{option?.points})</span>
+                    {allowBonus && (
+                      <button
+                        type="button"
+                        className="removeBonusButton"
+                        onClick={() => handleBonusRemove(bonusId)}
+                        title="Remove bonus"
+                      >
+                        ×
+                      </button>
+                    )}
+                  </div>
+                );
+              })}
+              <div className="bonusTotalDisplay">
+                Total Bonus: {currentBonus} points
+              </div>
+            </div>
+          ) : (
+            <div className="noBonusDisplay">
+              {allowBonus ? "No bonuses selected" : "No bonus available (bid not met)"}
+            </div>
```

Comment:
Let's tighten this UI up a bit. Rather than a completely separate line for "Selected bonuses: and the bonus score, let's put it on a single line:

Selected Bonuses ([bonsusScore]) / No Bonus / Bid not met

Reply:
Getting close. I would like this new information to be above the list of bonuses (and don't put it in a box)

List box
Bonus Score information
List of selected bonuses